### ISSUE TITLE
link

### ISSUE THREAD

**Issue by treetrunkz** [rbhatia46/React-Portfolio#5]
The js logic to open up images in modal in portfolio section, #modal-01 and portfolio images is not included? correct me if I am mistaken. The modal images are saved in modal and start with the letter "m-" not in init.js, or portfolio.js

**Comment by WarriorofZarona** [comment]
I'd like to know this too. I tried figuring it out on my own, but I can't seem to get it right.

**Comment by treetrunkz** [comment]
I actually did figure this out (for the most part) the modal-1 is tied to the modernizr modal and it is assumed in the js that whichever image name matches the name of the modal title will be chosen. Using modernizer as the modal in this portfolio is a little disabling. If you can write your own modal logic I would recommend it. You can add my github / send me a message if you would like to contact

**Comment by WarriorofZarona** [comment]
Yes, is there a way we can connect? Or maybe you can share the code that you used?

I'm still a bit new at coding but I'd be able to pick it up if I had some kind of guidance about what to do. I appreciate the help!

**Comment by davidlindercodes** [comment]
@treetrunkz  Yes the portfolio images are not provided, you need to upload your own. I used ibb.co to host mine for free.

**Comment by davidlindercodes** [comment]
@WarriorofZarona if you are looking to see some example code, you can have a look at mine here: 
https://github.com/davidlindercodes/React-Portfolio
